ouroboros-consensus/test-consensus/Test/Dynamic/Network.hs

```diff
@@ -418,9 +417,8 @@ broadcastNetwork registry testBtime numCoreNodes nodeJoinPlan pInfo initRNG slot
                 CoreId  i -> joinSlotOf (CoreNodeId i)
                 RelayId _ -> error "broadcastNetwork: unexpected RelayId"
 
-      void $ forkLinked         registry $ niWithServerNode ni wait
-      void $ forkLinkedTransfer registry $ \registry' -> txProducer
-        registry'
```

Comment:
When Edsko spotted this, I suggested the same change (I first wrote this line before I wrote the actual `txProducer`). He wants to keep it in, because the fact that `txProducer` forks is an implementation detail. Additionally, this is the main use of `forkLinkedTransfer`, IIRC.

Reply:
I'm dropping this part of the diff.